### PR TITLE
test: Create new tests for Listing supported apps Features

### PR DESCRIPTION
**Pull request by VDT1805**
- Added rspec gem
- Include routing test and request test for Listing supported apps feature

### CODE REVIEW

File: spec/requests/apps_spec.rb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  let(:valid_attributes) {
+    # skip("Add a hash of attributes valid for your model")
+    {
+        name: "Datadog Test"
+    }
+  }
+
+  # let(:invalid_attributes) {
+  #   skip("Add a hash of attributes invalid for your model")
+  # }
```
Comment (xuantien-raksul) [inline spec/requests/apps_spec.rb:29]: Let's remove all unnecessary comments

File: spec/models/app_spec.rb
Comment (xuantien-raksul) [inline spec/models/app_spec.rb:1]: Let's remove this empty test file

**Comment by xuantien-raksul** [comment]
Let's enforce consistent coding style using a ruby code linter. Example: https://github.com/rubocop/rubocop

File: spec/models/app_spec.rb
Comment (VDT1805) [reply spec/models/app_spec.rb:1]: Done

File: spec/requests/apps_spec.rb (hunk 1 above)
Comment (VDT1805) [reply spec/requests/apps_spec.rb]: Done

**Comment by VDT1805** [comment]
Already done, thank you.

**Pull request merged**